dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/trackedentity/HibernateTrackedEntityStore.java

```diff
@@ -594,13 +594,9 @@ private String getFromSubQueryJoinOrderByAttributes(TrackedEntityQueryParams par
    * Generates an INNER JOIN for program owner. This segment is only included if program is
    * specified or user is not super.
```

Comment:
> This segment is only included if program is
   * specified or user is not super.

Is this sentence still true?

Reply:
nope, that part is not true anymore